File: app/src/main/java/br/com/unicid/aplicativodentista/database/UsuarioDAO.java
package br.com.unicid.aplicativodentista.database;

import android.content.ContentValues;
import android.content.Context;
import android.database.sqlite.SQLiteDatabase;

import br.com.unicid.aplicativodentista.model.Usuario;

public class UsuarioDAO {

    private UsuarioDatabase usuarioDB;
    private SQLiteDatabase banco;

    public UsuarioDAO(Context context) {
        usuarioDB = new UsuarioDatabase(context);
        banco = usuarioDB.getWritableDatabase();
    }

    public long salvar(Usuario usuario) {
        ContentValues values = new ContentValues();
        values.put("NOME", usuario.getNome());
        values.put("CPF", usuario.getCPF());
        values.put("CRO", usuario.getCRO());
        values.put("EMAIL", usuario.getEmail());
        values.put("SENHA", usuario.getSenha());
        return banco.insert("USUARIO", null, values);
    }
}

File: app/src/main/java/br/com/unicid/aplicativodentista/dao/UsuarioDAO.kt
package br.com.unicid.aplicativodentista.dao

import android.content.ContentValues
import android.content.Context
import android.database.sqlite.SQLiteDatabase
import br.com.unicid.aplicativodentista.database.UsuarioDatabase
import br.com.unicid.aplicativodentista.model.Usuario

class UsuarioDAO(context: Context?) {
    private val usuarioDB: UsuarioDatabase
    private val banco: SQLiteDatabase

    init {
        usuarioDB = UsuarioDatabase(context!!)
        banco = usuarioDB.writableDatabase
    }

    fun salvarUsuario(usuario: Usuario): Long {
        val values = ContentValues()
        values.put("NOME", usuario.nome)
        values.put("CPF", usuario.cpf)
        values.put("CRO", usuario.cro)
        values.put("EMAIL", usuario.email)
        values.put("SENHA", usuario.senha)
        return banco.insert("USUARIO", null, values)
    }
}